core/src/main/scala/org/apache/spark/storage/ShuffleBlockFetcherIterator.scala

```diff
@@ -160,6 +160,9 @@ final class ShuffleBlockFetcherIterator(
   @GuardedBy("this")
   private[this] val shuffleFilesSet = mutable.HashSet[DownloadFile]()
 
+  private[this] val onCompleteCallback: ShuffleFetchCompletionListener =
```

Comment:
we can omit the type declaration since it's private val

Reply:
Done! thanks!